### DISCUSSION TITLE
0.13 release?

### DISCUSSION THREAD

**Discussion by raybellwaves** [anitagraser/movingpandas#265] [General]
@anitagraser could you do a 0.13 release please?

```
❯ git log $(git describe --tags --abbrev=0)..HEAD --oneline
ebd035c (HEAD -> main, origin/main, origin/HEAD) Make plotting dependencies optional, fixes #78
a471e1a Merge branch 'main' of https://github.com/anitagraser/movingpandas
3a4a36b Fix binder links: rename master to main
4eb8bdc Fix binder links: rename master to main
69e220e Rename master to main
2f1c013 update pre-commit, fixes #256
22b45bb Rename master to main
f35426c Change badge color
```

Behind the scenes:

Made some interactive plotting libraries optional (#261 )
Fixed broken pre-commit (#257 )

**Comment by anitagraser** [comment]
Yes, I'll look into it tomorrow

**Comment by anitagraser** [comment]
https://github.com/anitagraser/movingpandas/releases/tag/v0.13

**Comment by anitagraser** [comment]
@raybellwaves  Will you deal with readthedocs? https://readthedocs.org/projects/movingpandas/builds/18801101/

**Reply by raybellwaves** [reply]
we can move convo to https://github.com/anitagraser/movingpandas/issues/262

**Comment by anitagraser** [comment]
Looks like the example notebooks are working with 0.13 so everything besides the docs should be set
